gslb/gslbutils/gslbutils.go

```diff
@@ -673,3 +679,18 @@ func SetGslbConfigObjUpdated(value bool) {
 func GetGslbConfigObjUpdated() bool {
 	return gslbConfigObjUpdated
 }
+
+// Difference compares two slices a & b, returns the elements in `a` that aren't in `b`.
+func SliceDifference(a, b []string) []string {
```

Comment:
`set` DS has similar functionality.
refer to the file vendor/k8s.io/apimachinery/pkg/util/sets/string.go for more info.